### ISSUE TITLE
Rhino crash when importing

### ISSUE THREAD

**Issue by ThanePatrol** [DerLando/RhinoBridge#1]
Rhino crashes immediately upon import of a quixel asset, no error code.

All views were in Wireframe mode prior to importing. Asset was a 4k plant model

Other plugins are Vray and Lands-design

Rhino version : 7.4.21078.1001
Quixel bridge: 2021.0.0
RhinoBridge: 0.21
Vray: 5.10.02
Lands-design: 5.4.0.6748

**Comment by DerLando** [comment]
Yeah I'm sure the Rhino 7 API has changed from it's WIP state and I never got around to updating the PlugIn.

I hope I will have time to look into this at the weekend.

I should for sure push a minor release rather soon to introduce more meaningful error messages.

Thanks for reporting, I'll keep you updated

**Comment by rdcvoliveira** [comment]
Hey! I'm having the same problem when importing into rhino. It just crashes.... Is there any news about this issue?

**Comment by rdcvoliveira** [comment]
btw my rhino is: Rhino 7, 7.0.20314.03001
and my bridge is the latest: 2022.0.6
